### PR TITLE
Update README.md Template with BridgeCrew Compliance Badges

### PR DESCRIPTION
**Pull request by maximmi**
## what
* README.md Template updated with BridgeCrew Compliance Badges

## why
* To be able see and fix the recommendations from Bridgecrew so we can position our modules as standards compliant

### CODE REVIEW

File: templates/README.md.gotmpl [hunk 1]
```diff
@@ -83,6 +83,21 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
 
 {{ (ds "config").introduction -}}
 {{ end }}
+
```
Comment (osterman) [inline templates/README.md.gotmpl:88]: We need to wrap this in a check for *.tf files and only include it then

**Review by osterman — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -62,6 +62,21 @@ It's 100% Open Source and licensed under the [APACHE2](LICENSE).
 
 
 
+## Compliance
```
Comment (osterman) [inline README.md]: The `build-harness` should not have a `## Compliance` section. See comment: https://github.com/cloudposse/build-harness/pull/256/files#r524569945

File: templates/README.md.gotmpl [hunk 3]
```diff
@@ -83,6 +83,21 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
...
+## Compliance
```
Comment (osterman) [inline templates/README.md.gotmpl]:
```suggestion
## Security & Compliance [<img src="https://cloudposse.com/wp-content/uploads/2020/11/bridgecrew.svg" width="250" align="right" />](https://bridgecrew.io/)

Security scanning is graciously provided by Bridgecrew. Bridgecrew is the leading fully hosted, cloud-native solution providing continuous Terraform security and compliance.
```

File: templates/README.md.gotmpl [hunk 4]
```diff
@@ -83,6 +83,21 @@ We literally have [*hundreds of terraform modules*][terraform_modules] that are
...
+| [![SOC2]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/soc2" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=SOC2" (ds "config").github_repo | regexp.Replace "/" "%2F"}})| Service Organization Control 2 Compliance |
+| [![CIS GCP]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/cis_gcp" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=CIS+GCP+V1.1" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Center for Internet Security, GCP Compliance |
+| [![HIPAA]({{ printf "https://www.bridgecrew.cloud/badges/github/%s/hipaa" (ds "config").github_repo}})]({{ printf "https://www.bridgecrew.cloud/link/badge?vcs=github&fullRepo=%s&benchmark=HIPAA" (ds "config").github_repo | regexp.Replace "/" "%2F"}}) | Health Insurance Portability and Accountability Compliance |
```
Comment (osterman) [inline templates/README.md.gotmpl]: Replacing `/` with `%2f` is creating invalid URLs (E.g. `https:%2f%2fwww.bridgrcrew....`)

File: templates/README.md.gotmpl (hunk 1 above)
Comment (maximmi) [reply templates/README.md.gotmpl:88]: done

File: README.md (hunk 2 above)
Comment (maximmi) [reply README.md]: done

File: templates/README.md.gotmpl (hunk 4 above)
Comment (maximmi) [reply templates/README.md.gotmpl]: fixed

**Review by osterman — APPROVED** [review]

**Pull request merged**